src/Frontend/Panes/Lobbies/ConfigurationPane.tsx

```diff
@@ -54,215 +55,104 @@ const panes: ReadonlyArray<PaneConfig> = [
   {
     title: 'Game settings',
     shortcut: `1`,
-    path: '/settings/game',
+    path: '/game',
```

Comment:
why update path for all of these?

Reply:
left over from when i was messing with routing, but i think its simpler